Tiltfile

```diff
@@ -25,21 +25,6 @@ default_registry(settings.get("default_registry"))
 always_enable_providers = ["core"]
 
 providers = {
-    "core": {
-        "context": ".",
-        "image": "gcr.io/k8s-staging-cluster-api/cluster-api-controller",
-        "live_reload_deps": [
-            "main.go",
-            "go.mod",
-            "go.sum",
-            "api",
-            "cmd",
-            "controllers",
-            "errors",
-            "third_party",
-            "util",
-        ],
-    },
     "kubeadm-bootstrap": {
```

Comment:
Wouldn't we also need to pass in the feature gate to the `kubeadm-bootstrap` provider as well?

Reply:
yep, if this is agreeable overall I can shuffle those over too. Sounds like we can probably pull from tilt-settings and I'll revert the tilt-provider changes